runtime/include/chpl-privatization.h

```diff
@@ -23,10 +23,18 @@
 #include <stdint.h>
 #include "chpltypes.h"
 
+extern void** chpl_privateObjects;
+
```

Comment:
I'd recommend putting this extern declaration inside the function chpl_getPrivatizedClass() just to keep the private objects only accessible through the getter.